firmware/LEDArray.cpp

```diff
@@ -42,17 +42,21 @@ void serialEvent() {
 #elif defined(LED_ARRAY_MASTER)
 
 #include "../SerializerUtils.h"
+#include "../LEDArrayConnection.h"
```

Comment:
Do you think we should put all the .h files into one directory and compile it with the path of that directory so that we don't need to have relative path to the file?